src/codeocean/data_asset.py

```diff
@@ -51,6 +52,19 @@ class AppParameter:
     value: str
 
 
+@dataclass_json
+@dataclass(frozen=True)
+class ResultsInfo:
+    capsule_id: Optional[str] = None
+    pipeline_id: Optional[str] = None
+    version: Optional[int] = None
+    commit: Optional[str] = None
+    run_script: Optional[str] = None
+    data_assets: Optional[list[str]] = None
+    parameters: Optional[list[Param]] = None
+    processes: Optional[list[PipelineProcessParams]] = None
```

Comment:
The spec specifies here `PipelineProcess`, not `PipelineProcessParams`

Reply:
Corrected.